app/src/main/java/org/jellyfin/androidtv/integration/LeanbackChannelWorker.kt

```diff
@@ -130,6 +130,7 @@ class LeanbackChannelWorker(
 	/**
 	 * Updates the "my media" row with current media libraries.
 	 */
+	@Suppress("RestrictedApi")
```

Comment:
Could one of the annotations in this doc be used instead to conditionally enable/disable the lean back channel stuff?

The doc says when it's eventually (and maybe already?) blocked in new versions of android, it could cause breakage.

https://developer.android.com/about/versions/10/non-sdk-q#code-annotations

Reply:
The used API's are all from androidx.tvprovider. They shouldn't be restriced, just Google not maintaining their libraries for Android TV like always.
The linked annotations are meant for functions that use the platform API under the hood, tvprovider does that but detects if the platform functions are available and does nothing as fallback (so no crashes).

The reason I added the suppress annotations in this PR is because this version of the Gradle plugin added new (or improved existing) linter issues to find usages of restricted API's. If usage of these functions would break something it should've happened when we changes the Android SDK target in #1299.